src/rom_operator_inference/lstsq/_tikhonov.py

```diff
@@ -145,24 +145,25 @@ class SolverL2(_BaseTikhonovSolver):
         min_{X} ||AX - B||_F^2 + ||λX||_F^2,                    λ ≥ 0.
     """
     # Validation --------------------------------------------------------------
-    def _process_regularizer(self, λ):
-        """Validate the regularization hyperparameter and return λ^2."""
-        if not np.isscalar(λ):
-            raise TypeError("regularization hyperparameter λ must be "
+    def _process_regularizer(self, lambda_par):
+        """Validate the regularization hyperparameter and return
+        lambda_par^2."""
+        if not np.isscalar(lambda_par):
+            raise TypeError("regularization hyperparameter lambda_par must be "
                             "a scalar")
-        if λ < 0:
-            raise ValueError("regularization hyperparameter λ must be "
-                             "non-negative")
-        return λ**2
+        if lambda_par < 0:
+            raise ValueError("regularization hyperparameter lambda_par must be"
+                             " non-negative")
+        return lambda_par**2
 
     # Helper methods ----------------------------------------------------------
-    def _Σinv(self, λ):
+    def _sigma_inv(self, lambda_par):
```

Comment:
Another verbose/nonGreek name would be better here, maybe `_invsvals()` instead of `_sigma_inv()` (and later, `self._svals` instead of `self._sigma`)?